tokens/0x2ef27bf41236bd859a95209e17a43fbd26851f92.yaml

```diff
@@ -0,0 +1,19 @@
+---
+addr: '0x7b0c06043468469967dba22d1af33d77d44056c8'
```

Comment:
Looks like this contract address actually belongs to TDH Trustedhealth

Reply:
Correction:
0x2ef27bf41236bd859a95209e17a43fbd26851f92 = Old MORPH
0x7b0c06043468469967dba22d1af33d77d44056c8 = New MORPH
0x2a1dbabe65c595b0022e75208c34014139d5d357 = TDH Trustedhealth

The file for TDH Trustedhealth appears to have been erroneously renamed.

Reply:
OMG